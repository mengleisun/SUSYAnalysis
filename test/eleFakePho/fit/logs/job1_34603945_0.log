000 (34603945.000.000) 2021-09-20 06:02:57 Job submitted from host: <131.225.188.57:9618?addrs=131.225.188.57-9618&alias=lpcschedd2.fnal.gov&noUDP&sock=schedd_2251_93c6>
...
040 (34603945.000.000) 2021-09-20 06:03:22 Started transferring input files
	Transferring to host: <131.225.206.189:9618?addrs=131.225.206.189-9618&alias=cmswn1814.fnal.gov&noUDP&sock=slot1_7_1219_a54b_4449>
...
040 (34603945.000.000) 2021-09-20 06:03:22 Finished transferring input files
...
001 (34603945.000.000) 2021-09-20 06:03:22 Job executing on host: <131.225.206.189:9618?addrs=131.225.206.189-9618&alias=cmswn1814.fnal.gov&noUDP&sock=startd_1133_b8e4>
...
006 (34603945.000.000) 2021-09-20 06:03:31 Image size of job updated: 14
	14  -  MemoryUsage of job (MB)
	15088  -  ResidentSetSize of job (KB)
...
006 (34603945.000.000) 2021-09-20 06:04:31 Image size of job updated: 244
	244  -  MemoryUsage of job (MB)
	250000  -  ResidentSetSize of job (KB)
...
006 (34603945.000.000) 2021-09-20 06:05:32 Image size of job updated: 254
	254  -  MemoryUsage of job (MB)
	260528  -  ResidentSetSize of job (KB)
...
006 (34603945.000.000) 2021-09-20 06:05:58 Image size of job updated: 422
	422  -  MemoryUsage of job (MB)
	432956  -  ResidentSetSize of job (KB)
...
040 (34603945.000.000) 2021-09-20 06:05:58 Started transferring output files
...
040 (34603945.000.000) 2021-09-20 06:05:58 Finished transferring output files
...
005 (34603945.000.000) 2021-09-20 06:06:00 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:01:32, Sys 0 00:00:07  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:01:32, Sys 0 00:00:07  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	92098  -  Run Bytes Sent By Job
	1845500  -  Run Bytes Received By Job
	92098  -  Total Bytes Sent By Job
	1845500  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     0.00        1         1 
	   Disk (KB)            :  8236        2000    319815 
	   Memory (MB)          :   422        2048      2048 

	Job terminated of its own accord at 2021-09-20T11:05:58Z.
...
