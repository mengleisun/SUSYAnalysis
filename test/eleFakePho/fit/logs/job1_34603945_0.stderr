WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
rm: cannot remove 'RunnerScript.txt': No such file or directory
In file included from FitKer.C:50:0:
../../../include/analysis_tools.h: In function 'double getEvtWeight(int, int, double)':
../../../include/analysis_tools.h:34:12: warning: suggest parentheses around assignment used as truth value [-Wparentheses]
     if(year=2016) luminosity = 35921.875595;
        ~~~~^~~~~
../../../include/analysis_tools.h:35:12: warning: suggest parentheses around assignment used as truth value [-Wparentheses]
     if(year=2017) luminosity = 41529.548819;
        ~~~~^~~~~
../../../include/analysis_tools.h:36:12: warning: suggest parentheses around assignment used as truth value [-Wparentheses]
     if(year=2018) luminosity = 59740.565202;
        ~~~~^~~~~
In file included from FitKer.C:52:0:
../../../include/tdrstyle.C: In function 'void CMS_lumi(TPad*, int)':
../../../include/tdrstyle.C:59:9: warning: unused variable 'H' [-Wunused-variable]
   float H = pad->GetWh();
         ^
../../../include/tdrstyle.C:60:9: warning: unused variable 'W' [-Wunused-variable]
   float W = pad->GetWw();
         ^
FitKer.C: In function 'void FitKer(int, int, int, float, float, int, int)':
FitKer.C:138:138: warning: format '%s' expects argument of type 'char*', but argument 2 has type 'std::__cxx11::string* {aka std::__cxx11::basic_string<char>*}' [-Wformat=]
    etree->Add(Form("root://cmseos.fnal.gov//store/user/tmishra/elefakepho/files/plot_elefakepho_%sTnP_dR05_2016_probe35.root",processName));
                                                                                                                                          ^
FitKer.C:140:138: warning: format '%s' expects argument of type 'char*', but argument 2 has type 'std::__cxx11::string* {aka std::__cxx11::basic_string<char>*}' [-Wformat=]
    etree->Add(Form("root://cmseos.fnal.gov//store/user/tmishra/elefakepho/files/plot_elefakepho_%sTnP_dR05_2017_probe35.root",processName));
                                                                                                                                          ^
FitKer.C:142:138: warning: format '%s' expects argument of type 'char*', but argument 2 has type 'std::__cxx11::string* {aka std::__cxx11::basic_string<char>*}' [-Wformat=]
    etree->Add(Form("root://cmseos.fnal.gov//store/user/tmishra/elefakepho/files/plot_elefakepho_%sTnP_dR05_2018_probe35.root",processName));
                                                                                                                                          ^
FitKer.C:486:18: warning: unused variable 'toysignalRes' [-Wunused-variable]
   RooFFTConvPdf *toysignalRes;
                  ^~~~~~~~~~~~
FitKer.C:92:14: warning: variable 'useCMSShape' set but not used [-Wunused-but-set-variable]
         bool useCMSShape;
              ^~~~~~~~~~~
rm: cannot remove 'configFit_Bw_ker.txt': No such file or directory
rm: cannot remove 'configFit_DY_ker.txt': No such file or directory
rm: cannot remove 'configFit_Bw_expo.txt': No such file or directory
Unable to find user certificate or key: /home/tmishra/.globus/usercred.p12
Error in <TNetXNGFile::Open>: [ERROR] Server responded with an error: [3011] Unable to open file /eos/uscms/store/user/tmishra/elefakepho/files/plot_elefakepho_5û4ÿTnP_dR05_2017_probe35.root; No such file or directory

Warning in <TH1F::Sumw2>: Sum of squares of weights structure already created
Error in <ROOT::Math::FitResult>: FitConfiguration and Minimizer result are not consistent
Info in <TCanvas::Print>: png file DY_DY_ker_pt_num_60-120_40_45.png has been created

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================

Thread 6 (Thread 0x7f790af79700 (LWP 337)):
#0  0x00007f791b104d19 in syscall () from /lib64/libc.so.6
#1  0x00007f790d25579e in XrdSys::LinuxSemaphore::Wait (this=0x34fd8b0) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/./XrdSys/XrdSysLinuxSemaphore.hh:161
#2  XrdCl::SyncQueue<XrdCl::JobManager::JobHelper>::Get (this=0x3524478) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/./XrdCl/XrdClSyncQueue.hh:67
#3  XrdCl::JobManager::RunJobs (this=0x3524460) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdCl/XrdClJobManager.cc:146
#4  0x00007f790d255989 in RunRunnerThread (arg=<optimized out>) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdCl/XrdClJobManager.cc:33
#5  0x00007f791b3e1ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00007f791b10a9fd in clone () from /lib64/libc.so.6

Thread 5 (Thread 0x7f790b77a700 (LWP 336)):
#0  0x00007f791b104d19 in syscall () from /lib64/libc.so.6
#1  0x00007f790d25579e in XrdSys::LinuxSemaphore::Wait (this=0x34fd8b0) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/./XrdSys/XrdSysLinuxSemaphore.hh:161
#2  XrdCl::SyncQueue<XrdCl::JobManager::JobHelper>::Get (this=0x3524478) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/./XrdCl/XrdClSyncQueue.hh:67
#3  XrdCl::JobManager::RunJobs (this=0x3524460) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdCl/XrdClJobManager.cc:146
#4  0x00007f790d255989 in RunRunnerThread (arg=<optimized out>) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdCl/XrdClJobManager.cc:33
#5  0x00007f791b3e1ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00007f791b10a9fd in clone () from /lib64/libc.so.6

Thread 4 (Thread 0x7f790bf7b700 (LWP 335)):
#0  0x00007f791b104d19 in syscall () from /lib64/libc.so.6
#1  0x00007f790d25579e in XrdSys::LinuxSemaphore::Wait (this=0x34fd8b0) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/./XrdSys/XrdSysLinuxSemaphore.hh:161
#2  XrdCl::SyncQueue<XrdCl::JobManager::JobHelper>::Get (this=0x3524478) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/./XrdCl/XrdClSyncQueue.hh:67
#3  XrdCl::JobManager::RunJobs (this=0x3524460) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdCl/XrdClJobManager.cc:146
#4  0x00007f790d255989 in RunRunnerThread (arg=<optimized out>) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdCl/XrdClJobManager.cc:33
#5  0x00007f791b3e1ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00007f791b10a9fd in clone () from /lib64/libc.so.6

Thread 3 (Thread 0x7f790c77c700 (LWP 334)):
#0  0x00007f791b3e8e9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00007f790d61bded in XrdSysTimer::Wait (mills=<optimized out>) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdSys/XrdSysTimer.cc:239
#2  0x00007f790d1f9118 in XrdCl::TaskManager::RunTasks (this=0x350fc90) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdCl/XrdClTaskManager.cc:244
#3  0x00007f790d1f9269 in RunRunnerThread (arg=<optimized out>) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdCl/XrdClTaskManager.cc:37
#4  0x00007f791b3e1ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00007f791b10a9fd in clone () from /lib64/libc.so.6

Thread 2 (Thread 0x7f790cf7d700 (LWP 333)):
#0  0x00007f791b10afd3 in epoll_wait () from /lib64/libc.so.6
#1  0x00007f790d620c17 in XrdSys::IOEvents::PollE::Begin (this=0x34f8dd0, syncsem=<optimized out>, retcode=<optimized out>, eTxt=<optimized out>) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/./XrdSys/XrdSysIOEventsPollE.icc:213
#2  0x00007f790d61d4f5 in XrdSys::IOEvents::BootStrap::Start (parg=0x7fff34fa8630) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdSys/XrdSysIOEvents.cc:131
#3  0x00007f790d61b5e8 in XrdSysThread_Xeq (myargs=0x35071b0) at /build/cmsbld/auto-builds/CMSSW_10_2_0_pre6-slc7_amd64_gcc700/build/CMSSW_10_2_0_pre6-build/BUILD/slc7_amd64_gcc700/external/xrootd/4.8.3-gnimlf/xrootd-4.8.3/src/XrdSys/XrdSysPthread.cc:86
#4  0x00007f791b3e1ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00007f791b10a9fd in clone () from /lib64/libc.so.6

Thread 1 (Thread 0x7f7918068c40 (LWP 309)):
#0  0x00007f791b0d1549 in waitpid () from /lib64/libc.so.6
#1  0x00007f791b04ef62 in do_system () from /lib64/libc.so.6
#2  0x00007f791b04f311 in system () from /lib64/libc.so.6
#3  0x00007f791d584acb in TUnixSystem::StackTrace() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_2_22/external/slc7_amd64_gcc700/lib/libCore.so
#4  0x00007f791d58710c in TUnixSystem::DispatchSignals(ESignals) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_2_22/external/slc7_amd64_gcc700/lib/libCore.so
#5  <signal handler called>
#6  0x00007f791dd30fb8 in RooMCStudy::run(bool, bool, int, int, bool, char const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_2_22/external/slc7_amd64_gcc700/lib/libRooFitCore.so
#7  0x00007f791dd31c4d in RooMCStudy::generateAndFit(int, int, bool, char const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_2_22/external/slc7_amd64_gcc700/lib/libRooFitCore.so
#8  0x000000000041b4a0 in FitKer (fitFunc=2, inputbintype=0, inputfittype=1, lowercut=40, uppercut=45, fitrangelow=60, fitrangehigh=120) at FitKer.C:478
#9  0x000000000041da4f in main (argc=8, argv=0x7fff34fb89a8) at FitKer.C:591
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum http://root.cern.ch/forum.
Only if you are really convinced it is a bug in ROOT then please submit a
report at http://root.cern.ch/bugs. Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#6  0x00007f791dd30fb8 in RooMCStudy::run(bool, bool, int, int, bool, char const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_2_22/external/slc7_amd64_gcc700/lib/libRooFitCore.so
#7  0x00007f791dd31c4d in RooMCStudy::generateAndFit(int, int, bool, char const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_2_22/external/slc7_amd64_gcc700/lib/libRooFitCore.so
#8  0x000000000041b4a0 in FitKer (fitFunc=2, inputbintype=0, inputfittype=1, lowercut=40, uppercut=45, fitrangelow=60, fitrangehigh=120) at FitKer.C:478
#9  0x000000000041da4f in main (argc=8, argv=0x7fff34fb89a8) at FitKer.C:591
===========================================================


[9.062kB/9.062kB][100%][==================================================][9.062kB/s]  [9.062kB/9.062kB][100%][==================================================][9.062kB/s]  
Job: 1/4
Source: file://localhost/storage/local/data1/condor/execute/dir_24399/CMSSW_10_2_22/src/SUSYAnalysis/test/eleFakePho/fit/RunnerScript.txt
Target: root://cmseos.fnal.gov:1094//store/user/tmishra/elefakepho/DYFitting17/RunnerScript.txt
[7.939kB/7.939kB][100%][==================================================][7.939kB/s]  [7.939kB/7.939kB][100%][==================================================][7.939kB/s]  
Job: 2/4
Source: file://localhost/storage/local/data1/condor/execute/dir_24399/CMSSW_10_2_22/src/SUSYAnalysis/test/eleFakePho/fit/configFit_Bw_expo.txt
Target: root://cmseos.fnal.gov:1094//store/user/tmishra/elefakepho/DYFitting17/configFit_Bw_expo.txt
[56B/56B][100%][==================================================][56B/s]  [56B/56B][100%][==================================================][56B/s]  
Job: 3/4
Source: file://localhost/storage/local/data1/condor/execute/dir_24399/CMSSW_10_2_22/src/SUSYAnalysis/test/eleFakePho/fit/configFit_Bw_ker.txt
Target: root://cmseos.fnal.gov:1094//store/user/tmishra/elefakepho/DYFitting17/configFit_Bw_ker.txt
[56B/56B][100%][==================================================][56B/s]  [56B/56B][100%][==================================================][56B/s]  
Job: 4/4
Source: file://localhost/storage/local/data1/condor/execute/dir_24399/CMSSW_10_2_22/src/SUSYAnalysis/test/eleFakePho/fit/configFit_DY_ker.txt
Target: root://cmseos.fnal.gov:1094//store/user/tmishra/elefakepho/DYFitting17/configFit_DY_ker.txt
[56B/56B][100%][==================================================][56B/s]  [56B/56B][100%][==================================================][56B/s]  
rm: cannot remove 'EleFakeRate*.txt': No such file or directory
